components/expenses/Order.js

```diff
@@ -265,15 +296,61 @@ class Order extends React.Component {
           {order.status === 'PENDING' && canMarkOrderAsPaid && (
             <Flex>
               <MarkOrderAsPaidBtn order={order} collective={order.collective} />
-              <StyledButton bg="red.500" color="#fff" onClick={() => this.props.onClickCancel(order.id)}>
+              <StyledButton
+                bg="red.500"
+                color="#fff"
```

Comment:
Let's rather use `buttonStyle="danger"`, this doesn't look very good

![image](https://user-images.githubusercontent.com/1556356/71014732-b3446580-20f2-11ea-9de7-265333371ea1.png)